app/controllers/authController.js

```diff
@@ -75,6 +75,14 @@ class AuthController {
     const resMessage = await AuthService.resetPassword(newPassword, token);
     return ResponseHandler.success(res, resMessage);
   }
+
+  // send activate email to new user
+  async sendActivateEmail(req, res) {
```

Comment:
To the best of my knowledge, the controller layer are not responsible for additional business logic except param check and invoking the corresponding service. Is that better to refactor this logic to auth service? Need to be discussion!